master/master.cfg

```diff
@@ -315,18 +333,24 @@ for name, worker_name, buildfactory, stability, tier in BUILDERS:
     tags = ["PullRequest", stability, *getattr(f, "tags", [])]
     if tier:
         tags.append(tier)
-    c["builders"].append(
-        util.BuilderConfig(
-            name=buildername,
-            workernames=[worker_name],
-            builddir="%s.%s%s"
-            % ("pull_request", worker_name, getattr(f, "buildersuffix", "")),
-            factory=f,
-            tags=tags,
-            locks=[cpulock.access("counting")],
-        )
+
+    builder = util.BuilderConfig(
+        name=buildername,
+        workernames=[worker_name],
+        builddir="%s.%s%s"
+        % ("pull_request", worker_name, getattr(f, "buildersuffix", "")),
+        factory=f,
+        tags=tags,
+        locks=[cpulock.access("counting")],
     )
 
+    # This worker runs pyperformance at 12am. If a build is scheduled between
+    # 10pm and 2am, it will be delayed at 2am.
+    if worker_name == "diegorusso-aarch64-bigmem":
+        builder.nextBuild = no_builds_between(22, 2)
+
+    c["builders"].append(builder)
+
```

Comment:
Note that this section is specific to the PR builders; for regular builders look around line 250.  The same treatment will be needed there.

Reply:
OK, replicated the same logic in the previous section.